docs/integrations/github_app.md

```diff
@@ -0,0 +1,116 @@
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - INCOMPLETE WITH FAILURE
+
+![GitHub app PR comment - incomplete with failure](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_incomplete_failure.png)
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - INCOMPLETE
+
+![GitHub app PR comment - incomplete](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_incomplete.png)
+
+---
+
+Phylum OSS Supply Chain Risk Analysis - SUCCESS
+
+![GitHub app PR comment - success](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_pr_comment_success.png)
+
+---
+
+### Groups
+
+> **NOTE:** Only [Phylum PRO accounts](https://www.phylum.io/pricing) support groups.
+
+The GitHub App will automatically create a group with the name of your GitHub account/organization. All Phylum projects created by the GitHub App will be owned by that group and results can be shared by adding Phylum PRO accounts as members to the group.
+
+![Phylum groups management](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/phylum_groups_management.png)
+
+The default group that is created is owned by the account that installed the GitHub App. If you would like a different account to manage the GitHub App and group you may [transfer ownership](https://docs.phylum.io/docs/transfer_group_ownership) of the group.
+
+### Policy
+
+The Phylum GitHub app uses the [established project policy](https://docs.phylum.io/docs/policy) for making overall success/failure risk decisions. No configuration is required for setting the policy since a default policy is used for all projects. However, [Phylum PRO users](https://www.phylum.io/pricing) may specify custom policies for their projects to exercise fine-grained control over the risk decision logic.
+
+### Remediation
+
+There are several options to remediate failures determined by the GitHub app. A good first step is to click the link in the PR comment to "View this project in the Phylum UI":
+
+![GitHub app PR comment link to project](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_view_project_link.png)
+
+That will provide results and details for individual package issues. Each issue can be reviewed and suppressed if it is found to be a false positive, irrelevant, or otherwise acceptable to proceed:
+
+![Phylum app issue suppression](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/issue_suppression.png)
+
+Remaining issues may require lockfile modification to find alternative dependencies or otherwise remove the offending package.
+
+## FAQ
+
+1. Q: I activated monitoring, but it didn't run a scan. How do I get analysis results?
+   * A: Activating monitoring tells the app to start watching for commits in a PR that modify supported lockfiles. To see scan results, start committing code!
```

Comment:
```suggestion
   * A: Check to ensure the repository contains a supported lockfile.
```

The application has been updated to automatically scan the default branch when monitoring is activated.